src/components/RichTextEditor/helpers/forceToFocusElement.ts

```diff
@@ -13,7 +13,7 @@ const TABBABLE_ELEMENTS = [
     'embed',
     'audio[controls]',
     'video[controls]',
-    '[contenteditable]',
+    '[contenteditable="true"]',
```

Comment:
Not related to that PR

Reply:
I know, but I intentionally commited this as this will be needed for the Do&Dont as well. Would you prefer to open a separate PR for this?

Reply:
Yes would be cool if you open another PR for that so we have it clean